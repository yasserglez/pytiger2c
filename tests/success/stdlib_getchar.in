pytiger2c.py (PyTiger2C) 1.0
Copyright (C) 2009, 2010 Yasser González Fernández <ygonzalezfernandez@gmail.com>
Copyright (C) 2009, 2010 Ariel Hernández Amador <gnuaha7@gmail.com>
