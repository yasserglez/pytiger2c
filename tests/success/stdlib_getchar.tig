let
    var c: string := ""
in
    while 1 do ( 
        c := getchar();
        if size(c) = 0 then break;
	print(c)
    )
end
