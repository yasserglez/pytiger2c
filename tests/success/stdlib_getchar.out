pytiger2c.py (PyTiger2C) 1.0
Copyright (C) 2009, 2010 Yasser González Fernández <yglez@uh.cu>
Copyright (C) 2009, 2010 Ariel Hernández Amador <gnuaha7@uh.cu>
